assets/src/edit-story/app/media/utils/useFFmpeg.js

```diff
@@ -219,7 +216,7 @@ function useFFmpeg() {
    * @return {boolean} Whether transcoding is likely possible.
    */
   const canTranscodeFile = (file) =>
-    MEDIA_TRANSCODING_SUPPORTED_INPUT_TYPES.includes(file.type);
+    allowedAllowedTranscodableTypes.includes(file.type);
```

Comment:
`allowedAllowed`?

Reply:
Can we use `allowedTranscodableTypes` for now? Or even `allowedTranscodableMimeTypes`?

But `allowedAllowed` is just duplicated 🤔

Reply:
🤦 I thought you had a problem with the name. I didn't even notice the allowedAllowed. My brain would not let me see it. I will change. 

My point stands able the messy config.